src/NzbDrone.Core/MediaFiles/EpisodeImport/Manual/ManualImportService.cs

```diff
@@ -246,6 +246,7 @@ private List<ManualImportItem> ProcessFolder(string rootFolder, string baseFolde
 
                 if (files.Count() > 100)
                 {
+                    _logger.Warn("Found more than 100 files for multiple series or no series. Skipping parsing.");
```

Comment:
```suggestion
                    _logger.Warn("Unable to determine series from folder name and found more than 100 files. Skipping parsing");
```

Which should avoid any confusion that it's the file names that matter in this case.